mscore/metaedit.ui

```diff
@@ -19,72 +22,178 @@
+        <string>API-Level:</string>
        </property>
-       <property name="readOnly">
-        <bool>true</bool>
+       <property name="textInteractionFlags">
+        <set>Qt::NoTextInteraction</set>
        </property>
       </widget>
      </item>
-     <item row="1" column="0">
-      <widget class="QLabel" name="label_7">
+     <item row="0" column="1" colspan="6">
+      <widget class="QLabel" name="filePath">
+       <property name="sizePolicy">
+        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
+         <horstretch>0</horstretch>
+         <verstretch>0</verstretch>
+        </sizepolicy>
+       </property>
+       <property name="toolTip">
+        <string>Where the score is saved on your computer.</string>
+       </property>
+       <property name="whatsThis">
+        <string>Where the score is saved on your computer.</string>
+       </property>
+       <property name="accessibleDescription">
+        <string>Where the score is saved on your computer.</string>
+       </property>
+       <property name="frameShape">
+        <enum>QFrame::Panel</enum>
+       </property>
        <property name="text">
-        <string>MuseScore Version:</string>
+        <string>The document has not been saved yet.</string>
+       </property>
+       <property name="textInteractionFlags">
+        <set>Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
-     <item row="1" column="3">
-      <widget class="QLineEdit" name="revision">
-       <property name="enabled">
-        <bool>false</bool>
+     <item row="1" column="5">
+      <widget class="QLabel" name="level">
+       <property name="sizePolicy">
+        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
+         <horstretch>0</horstretch>
+         <verstretch>0</verstretch>
+        </sizepolicy>
        </property>
-       <property name="readOnly">
-        <bool>true</bool>
+       <property name="minimumSize">
+        <size>
+         <width>51</width>
+         <height>0</height>
+        </size>
+       </property>
+       <property name="toolTip">
+        <string>API-Level</string>
+       </property>
+       <property name="frameShape">
+        <enum>QFrame::Panel</enum>
+       </property>
+       <property name="text">
+        <string>No level</string>
+       </property>
+       <property name="textInteractionFlags">
+        <set>Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
-     <item row="1" column="2">
-      <widget class="QLabel" name="label_8">
+     <item row="1" column="1">
+      <widget class="QLabel" name="version">
+       <property name="sizePolicy">
+        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
+         <horstretch>0</horstretch>
+         <verstretch>0</verstretch>
+        </sizepolicy>
+       </property>
+       <property name="toolTip">
+        <string>The MuseScore version which last saved the score.</string>
```

Comment:
I was unsure about which vs that (reading some online resources confused me move. I would suggest rephrasing these in the passive voice, e.g. 
The MuseScore version with which the score was last saved.